src/Cms/Languages.php

```diff
@@ -24,12 +24,13 @@ class Languages extends Collection
 	/**
 	 * Creates a new collection with the given language objects
 	 *
-	 * @param array $objects `Kirby\Cms\Language` objects
 	 * @param null $parent
 	 * @throws \Kirby\Exception\DuplicateException
 	 */
-	public function __construct($objects = [], $parent = null)
-	{
+	public function __construct(
+		array $objects = [],
+		$parent = null
```

Comment:
```suggestion
		object|null $parent = null
```

Reply:
According to the doc block, we actually don't want this parameter to be set to anything